linchpin/provision/dummy.yml

```diff
@@ -3,27 +3,18 @@
 
 - name:  "schema check and Pre Provisioning Activities on topology_file"
   hosts: localhost
-  gather_facts: True
+  gather_facts: False
   roles:
     - { role: 'common' }
 
 - name:  "Provisioning resources based on resource group type"
   hosts: localhost
-  gather_facts: True
+  gather_facts: False
   roles:
     - { role: 'dummy' }
 
 - name: "Writing contents to file"
   hosts: localhost
-  gather_facts: True
+  gather_facts: False
```

Comment:
Be careful about setting `gather_facts: False`. I've run into more than one issue where having gather_facts whas helpful, especially during the gather_resources role as it actually uses some of the details for libvirt (possibly others).